INSTALL.md

```diff
@@ -188,12 +188,19 @@ bundle config --local build.mysql2 "--with-ldflags=-L/usr/local/opt/openssl/lib
 
 and run `bundle install` again.
 
-#### NPM
+#### Node packages
 
-Run [NPM](https://www.npmjs.com/) to install all the required Node modules:
+Install [Yarn](https://yarnpkg.com/) globally:
 
 ```bash
-npm install
+npm install -g yarn
```

Comment:
I'd suggest homebrew before npm (since this is targeted to MacOS). In any case, I would not suggest anything at all. Providing the link above and leaving the user decide would be better IMHO.
```suggestion
brew install yarn
```

Reply:
That's the reason why we ditched it in the first place. The brew package uses ocaml, licenses not quite open source friendly...

Reply:
For the production env I agree with you but for development it's inconsistent that we use this tool with other dependencies (see [here](https://github.com/3scale/porta/blob/master/INSTALL.md#dependencies)) but not with Yarn.

Reply:
Btw I think there was a confusion when we search for how to productize yarn
It does not use OCaml at all ... (since the beginning) it just have tests that queries the OCaml repository which lead to this confusion ....

yarn can be used to resolve OCaml repositories ... that is why the tests were there ....
But we did not invest much time on it and jump into the conclusion that OCaml is necessary
https://github.com/yarnpkg/yarn/commit/8be4eef75414db89875f5c64b71cf06783b4507a